renamePaths.js

```diff
@@ -0,0 +1,30 @@
+
+if (app.documents.length > 0) {
+  var bkDoc = app.activeDocument;
+
+  // check for breakmark template
+  if (bkDoc.name == "breakmark-template.ai") {
+    var allArtboards = bkDoc.artboards;
+    app.selection = null;
+
+    for (var i = 0; i < allArtboards.length; i++) {
+      currentName = allArtboards[i].name;
+      allArtboards.setActiveArtboardIndex(i);
+      bkDoc.selectObjectsOnActiveArtboard();
+
+      for (var j = 0; j < app.selection.length; j++) {
```

Comment:
Nesting loops can get confusing really quickly, when I get a chance I'm going to pull part of this into a separate function

Reply:
so we loop through each artboard, select all on the active one, do a thing on each of the selected objects, then move to the next artboard.